cda-gui/src/main.jsx

```
@@ -0,0 +1,54 @@
+// Routing
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { Link, createBrowserRouter, RouterProvider } from "react-router-dom";
+
+import { LinkProvider } from "@usace/groundwork";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+// Pages
+import Home from "./pages/Home";
+import NotFound from "./pages/NotFound";
+import SwaggerUI from "./pages/swagger-ui/index";
+import Regexp from "./pages/regexp/index";
+import DataQuery from "./pages/data-query";
+import Layout from "./components/Layout";
+
+// Styles
+import "@usace/groundwork/dist/style.css";
+import "./css/index.css";
+import ErrorFallback from "./pages/data-query/components/ErrorFallBack";
```

Importing from components/ErrorFallBack instead of components/ErrorFallback here -- throws an error.